package/batocera/core/batocera-splash/batocera-splash.mk

```diff
@@ -103,6 +103,7 @@ endef
 define BATOCERA_SPLASH_INSTALL_ROTATE_IMAGE
 	mkdir -p $(TARGET_DIR)/usr/share/batocera/splash
 	convert "$(BR2_EXTERNAL_BATOCERA_PATH)/package/batocera/core/batocera-splash/logo.png" -fill white -pointsize 30 -annotate +50+1020 "$(BATOCERA_SPLASH_TGVERSION)" -rotate -90 "${TARGET_DIR}/usr/share/batocera/splash/logo-version.png"
+	ln -s "logo-version.png" "${TARGET_DIR}/usr/share/batocera/splash/boot-logo.png"
```

Comment:
Should be a `ln -sf`

Reply:
To be more precise, something like:
`cd "${TARGET_DIR}/usr/share/batocera/splash" && ln -sf logo-version.png boot-logo.png`